cmd/vic-machine/delete/delete.go

```diff
@@ -61,6 +61,12 @@ func (d *Uninstall) Flags() []cli.Flag {
 			Usage:       "Compute resource path, e.g. /ha-datacenter/host/myCluster/Resources/myRP",
 			Destination: &d.ComputeResourcePath,
 		},
+		cli.StringFlag{
+			Name:        "name",
+			Value:       "",
+			Usage:       "The name of the Virtual Container Host",
```

Comment:
"The name of the Virtual Container Host to delete; requires that compute-resource argument be supplied"